src/ansys/mapdl/core/launcher.py

```diff
@@ -597,6 +595,8 @@ def launch_grpc(
 
     # removing spaces in cells
     command = " ".join(command_parm)
+    command_parm = command.split(" ")
+    command_parm.insert(0, f"{exec_file}")
 
```

Comment:
Very good health check and defensive coding!

Reply:
For the sake of completeness.. one could not be sure that the initial "command_parm" did not have any spaces in some of its list entries. So, by joining them and splitting again based on spacing, this issue is solved. Finally, inserting the executable as the first element of the list leaves you with the same expected behavior